cmd/kubelet/app/server.go

```diff
@@ -625,6 +625,9 @@ func CreateAPIServerClientConfig(s *options.KubeletServer) (*restclient.Config,
 		return nil, err
 	}
 
+	// Hardcode "kubelet" instead of os.Args[0] to differentiate kubelet requests
+	// in metrics.
```

Comment:
this still confuses me: `os.Args[0]` equals `"kubelet"`. The comment does not explain why we have to hard-code.

Reply:
corrected, thanks